Sources/ApolloSQLite/SQLiteNormalizedCache.swift

```diff
@@ -95,11 +96,30 @@ public final class SQLiteNormalizedCache {
     return try self.db.prepare(query).map { try parse(row: $0) }
   }
 
-  private func clearRecords() throws {
-    try self.db.run(records.delete())
-    if self.shouldVacuumOnClear {
-      try self.db.prepare("VACUUM;").run()
+  private func clearRecords(accordingTo policy: CacheClearingPolicy) throws {
+    switch policy._value {
+    case let .first(count):
+      let firstKRecords = records.select(self.id).order(self.id.asc).limit(count)
```

Comment:
a) Mostly out of curiosity, why `K` rather than `N`?
b) for what it's worth I find these types  of things a lot easier to read if they're on separate lines so that you can see each of the steps more granularly

Reply:
`Collection.dropFirst` uses the `k` parameter label terminology because AFAIK the "term of art" in the functional world of a "count"

My first instinct is to follow the Swift Standard Library for naming things.

I'm not strongly invested to any of this naming - if you want `N` to be used instead I'll make the change

Reply:
fair enough on it matching the standard lib, don't need to change the name.